Client/UserResearch/OnboardingUserResearch.swift

```diff
@@ -56,12 +56,12 @@ class OnboardingUserResearch {
         }
     }
     
-    func updateValue(value: Bool) {
+    func updateValue(onboardingScreenType: Bool) {
```

Comment:
this should take the enum argument (instead of the Bool)

Reply:
Ah, I get it. Changed to enum.